tap_xero/transform.py

```diff
@@ -61,3 +61,15 @@ def format_invoices(invoices):
     for invoice in invoices:
         if invoice.get('Date') == '/Date(0+0000)/':
             invoice['Date'] = '1970-01-01T00:00:00.000000Z'
+
+
+def format_journals(journals):
+    # NB: Xero sometimes formats the JournalDate as '/Date(0+0000)/' to
+    # indicate it is 0 milliseconds from the unix epoch. Convert this to a
+    # datetime that will be accepted by the transformer. This should not
+    # cause inconsitencies because the 'Date' is normally returned as an
+    # iso8601 string and this edge case causes it to be returned
+    # differently
+    for journal in journals:
+        if journal.get('JournalDate') == '/Date(0+0000)/':
+            journal['JournalDate'] = '1970-01-01T00:00:00.000000Z'
```

Comment:
Would it be better to refactor the above function format_invoices to accept a key parameter so you could use the same function in both places? The logic is the same except for the key.

Reply:
For now I would rather keep them separate because we may find we need to do other formatting for journals in the future and combining them now would mean we need to undo that. If we find a third situation in which this is happening then I say yes combine them.